plugins/parcel-transformer-markdown-html/lib/index.js

```diff
@@ -0,0 +1,91 @@
+// @ts-check
+const { Transformer } = require('@parcel/plugin');
+const { promisify } = require('@parcel/utils');
+const marked = require('marked');
+const hljs = require('highlight.js');
+const frontMatter = require('front-matter');
+const mustache = require('mustache');
+const path = require('path');
+
+const markedParse = promisify(marked.parse);
+
+module.exports = new Transformer({
+	async loadConfig({ config }) {
+		let configFile = await config.getConfig(['.scaffoldrc']);
+
+		if (configFile) {
```

Comment:
nit- if you put the config assertions above this line you can remove the `if(configFile){`

Reply:
reorganized a tad